langserve/api_handler.py

```diff
@@ -64,6 +64,10 @@
     EventSourceResponse = Any
 
 
+def _is_hosted() -> bool:
+    return os.environ.get("HOSTED_LANGSERVE_ENABLED", "false").lower() == "true"
+
```

Comment:
stylistic, but perhaps:

```
_IS_HOSTED = _is_hosted() 
```

and import the variable instead of the function in server?